usage/plugins/org.jboss.tools.usage/src/org/jboss/tools/usage/event/UsageReporter.java

```diff
@@ -306,7 +310,7 @@ protected boolean isPreferencesEnabled() {
 
 	protected EventSender getEventSender() {
 		if(eventSender==null) {
-			eventSender = new GoogleAnalyticsEventSender();
+			eventSender = new TelemetrySender();
```

Comment:
this is the main switch where we stop sending to google analytics but send to segment instead.